docs/data/date-pickers/getting-started/ComponentExplorerNoSnap.tsx

```diff
@@ -0,0 +1,239 @@
+    const Component = exportedElements[
+      exportedName
+    ] as unknown as React.JSXElementConstructor<{}>;
+
+    return (
+      <DemoItem label={getSubPackageFromExportedName(exportedName)}>
+        <Component />
+      </DemoItem>
+    );
+  });
+
+  const docPages = React.useMemo(() => {
+    const docPagesNames = Array.from(
+      new Set(
+        exportedNames.map((exportedName) =>
+          exportedName.replace(
+            /(Unstable_|Desktop|Mobile|Static|Next|SingleInput|MultiInput)/g,
+            '',
+          ),
+        ),
+      ),
+    );
+
+    return docPagesNames.map((docPageName) => ({
+      name: camelCaseToPascalCase(docPageName),
+      path: `/x/react-date-pickers/${camelCaseToKebabCase(docPageName)}/`,
+    }));
+  }, [exportedNames]);
+
+  return (
+    <Stack spacing={3} sx={{ width: '100%', py: 2 }}>
+      <Stack direction="row" spacing={2}>
+        <FormControl>
+          <InputLabel id="component-explorer-value-type-label">
+            Value type
+          </InputLabel>
+          <Select
+            label="Value type"
+            labelId="component-explorer-value-type-label"
+            sx={{ minWidth: 192 }}
+            value={state.valueType}
+            onChange={(event) =>
+              setState((prev) => ({
+                ...prev,
+                valueType: event.target.value as State['valueType'],
+              }))
+            }
+          >
+            <MenuItem value="date">Date</MenuItem>
+            <MenuItem value="time">Time</MenuItem>
+            <MenuItem value="dateTime">Date and Time</MenuItem>
+            <MenuItem value="dateRange">
+              Date Range <span className="plan-pro" />
+            </MenuItem>
+            <MenuItem value="timeRange">
+              Time Range <span className="plan-pro" />
+            </MenuItem>
+            <MenuItem value="dateTimeRange">
+              Date Time Range <span className="plan-pro" />
+            </MenuItem>
+          </Select>
+        </FormControl>
+        <FormControl>
+          <InputLabel id="component-explorer-family-label">Family</InputLabel>
+          <Select
+            label="Family"
+            labelId="component-explorer-family-label"
+            sx={{ minWidth: 164 }}
+            value={state.family}
+            onChange={(event) =>
+              setState((prev) => ({
+                ...prev,
+                family: event.target.value as State['family'],
+              }))
+            }
+          >
+            <MenuItem value="picker">Picker</MenuItem>
+            <MenuItem value="field">Field</MenuItem>
+            <MenuItem value="view">Calendar / Clock</MenuItem>
+          </Select>
+        </FormControl>
+      </Stack>
+      {exportedNames.length > 0 && (
+        <React.Fragment>
+          <Stack>
+            {docPages.map((docPage) => (
+              <Link href={docPage.path} rel="noopener" target="_blank">
+                Documentation {docPage.name}
```

Comment:
What do you think about putting the `Documentation` word after the component?
Also it would be nice to change the styles/layout to make anchor not take the whole container width, because now you can mistakenly open it by clicking in empty space. 🙈  
![Screenshot 2023-01-12 at 17 17 25](https://user-images.githubusercontent.com/4941090/212105485-79d788e1-8cc8-4a78-9c77-52e570cf0dc6.png)